tests/gold_tests/autest-site/conditions.test.ext

```diff
@@ -16,8 +16,27 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+import subprocess
+
 def HasOpenSSLVersion(self, version):
-    return self.EnsureVersion(["openssl","version"],min_version=version)
+    output = subprocess.check_output(
```

Comment:
Can the openssl version extraction be done by parsing traffic_layout's JSON output?

```
jsondata = json.loads(subprocess.check_output([.../traffic_layout','--versions', '--json']))
openssl_version = jsondata['openssl_str']
```